core/services/pipeline/task.bridge.go

```diff
@@ -54,32 +68,49 @@ func (t *BridgeTask) Run(ctx context.Context, meta JSONSerializable, inputs []Re
 		)
 	}
 
-	requestData := withMeta(t.RequestData, metaMap)
-	if t.IncludeInputAtKey != "" && len(inputs) > 0 {
-		requestData[t.IncludeInputAtKey] = inputs[0].Value
+	requestData = withMeta(requestData, metaMap)
+	if t.IncludeInputAtKey != "" {
+		logger.Warnw(`The "includeInputAtKey" parameter on Bridge tasks is deprecated. Please migrate to variable interpolation syntax as soon as possible (see CHANGELOG).`,
+			"task", t.DotID(),
+		)
+		if len(inputValues) > 0 {
+			requestData[string(includeInputAtKey)] = inputValues[0]
+		}
 	}
 
-	result = (&HTTPTask{
-		URL:         models.WebURL(url),
-		Method:      "POST",
-		RequestData: requestData,
-		// URL is "safe" because it comes from the node's own database
-		// Some node operators may run external adapters on their own hardware
-		AllowUnrestrictedNetworkAccess: MaybeBoolTrue,
-		config:                         t.config,
-	}).Run(ctx, meta, nil)
-	if result.Error != nil {
-		return result
+	// URL is "safe" because it comes from the node's own database
```

Comment:
Following on from a previous comment I made, I think we should have UntrustedURL and TrustedURL types